misc/dbt-materialize/dbt/adapters/materialize/impl.py

```diff
@@ -14,18 +14,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List, Optional
+from dataclasses import dataclass
+from typing import Any, List, Mapping, Optional
 
+from dbt.adapters.base.impl import AdapterConfig
 from dbt.adapters.materialize import MaterializeConnectionManager
 from dbt.adapters.materialize.relation import MaterializeRelation
 from dbt.adapters.postgres import PostgresAdapter
 from dbt.adapters.sql.impl import LIST_RELATIONS_MACRO_NAME
+from dbt.exceptions import RuntimeException
+
+
+@dataclass
+class MaterializeConfig(AdapterConfig):
+    materialize_cluster: Optional[str] = None
```

Comment:
I think this should just be `cluster`, rather than `materialize_cluster`? At least, it seems similar to the `unlogged` PostgreSQL-specific option, which is called just `unlogged` rather than `postgresql_unlogged`.

Reply:
Thank you, updated!